cl/users/models.py

```diff
@@ -283,10 +284,69 @@ class BackoffEvent(AbstractDateTimeModel):
         help_text="The next retry datetime for exponential backoff events.",
     )
 
+    @property
+    def under_waiting_period(self) -> bool:
+        """Does the backoff event is under waiting period?
+
+        :return bool: True if so, False if not.
+        """
+        if now() < self.next_retry_date:
+            return True
+        else:
+            return False
+
     def __str__(self) -> str:
         return f"Backoff event for {self.email_address}, next: {self.next_retry_date}"
 
 
+class Email(AbstractDateTimeModel):
```

Comment:
I found `Email` to be confusing since it can be an email address or an email message. Maybe call it `EmailMessage` instead?

Reply:
Hm, I see that `EmailMessage` is taken by Django already, so maybe we don't want to confuse that. Maybe `EmailSent` or `EmailStore` instead?

Reply:
I changed for `EmailSent`